src/ekf_localization_node.cpp

```diff
@@ -38,9 +38,12 @@ int main(int argc, char **argv)
 {
   ros::init(argc, argv, "ekf_navigation_node");
 
-  RobotLocalization::RosEkf ekf;
+  ros::NodeHandle nh;
+  ros::NodeHandle nh_priv("~");
 
-  ekf.run();
+  RobotLocalization::RosEkf ekf(nh, nh_priv);
+  ekf.initialize();
+  ros::spin();
 
-  return 0;
+  return EXIT_SUCCESS;
```

Comment:
This is standard c/c++: http://en.cppreference.com/w/cpp/utility/program/EXIT_status